docs/deployment/deployment-cluster.md

```diff
@@ -59,7 +59,7 @@ This aritcle introduces how to delopy the `Shenyu` gateway in cluster enviroment
 
 * modify `upstream` and `server` of configuration in `nginx.conf`.
 
-```conf
+```nginx
```

Comment:
no `conf` hightlight in prism.js (https://github.com/PrismJS/prism/tree/master/components)